_studio/mfx_lib/shared/src/mfx_h264_fei_vaapi.cpp

```diff
@@ -1775,17 +1774,29 @@ mfxStatus VAAPIFEIPAKEncoder::Destroy()
 {
     MFX_AUTO_LTRACE(MFX_TRACE_LEVEL_HOTSPOTS, "FEI::PAK::Destroy");
 
-    mfxStatus sts = MFX_ERR_NONE;
-
-    MFX_DESTROY_VABUFFER(m_statParamsId,     m_vaDisplay);
-    MFX_DESTROY_VABUFFER(m_statMVId,         m_vaDisplay);
-    MFX_DESTROY_VABUFFER(m_statOutId,        m_vaDisplay);
-    MFX_DESTROY_VABUFFER(m_codedBufferId[0], m_vaDisplay);
-    MFX_DESTROY_VABUFFER(m_codedBufferId[1], m_vaDisplay);
-
-    sts = VAAPIEncoder::Destroy();
+#if defined(_DEBUG)
```

Comment:
emmm. looks ugly:). https://en.cppreference.com/w/cpp/utility/tuple/ignore - can this help somehow?

Reply:
No, my idea was a bit different. Since there is no good way to react on error status in destructor, I just want to make those statuses visible for debug. ```std::ignore``` doesn't allow to look for assigned value